node/node.go

```diff
@@ -163,7 +163,6 @@ func (n *Node) Start(ctx context.Context) error {
 
 	// Build the socket based on the hostname/IP provided in the CLI
 	socket := string(core.MakeOperatorSocket(n.Config.Hostname, n.Config.DispersalPort, n.Config.RetrievalPort))
-	n.Logger.Info("Registering node with socket", "socket", socket)
```

Comment:
This is misleading: it will not register if `RegisterNodeAtStart` is false (which is the default).